src/components/gas/GasSpeedButton.js

```diff
@@ -173,9 +172,9 @@ const GasSpeedButton = ({
     () =>
       isNil(price) ||
       isEmpty(gasFeeParamsBySpeed) ||
-      isEmpty(selectedGasFee?.gasFee) ||
-      isSufficientGas === null,
-    [gasFeeParamsBySpeed, isSufficientGas, price, selectedGasFee]
+      isEmpty(selectedGasFee?.gasFee)[
+        (gasFeeParamsBySpeed, price, selectedGasFee)
+      ]
```

Comment:
what the heck is going on here? :) 

I guess it's missing comma here
```suggestion
      isEmpty(selectedGasFee?.gasFee),
    [gasFeeParamsBySpeed, price, selectedGasFee]
```

Reply:
wow thanks!